pkg/config/config.go

```diff
@@ -936,3 +936,12 @@ func (c *Config) Write() error {
 	}
 	return nil
 }
+
+// Reload reloads the configuration from containers.conf files
+func Reload() (*Config, error) {
+	newConfig, err := NewConfig("")
+	if err != nil {
+		return nil, errors.Wrapf(err, "containres.conf reload failed")
+	}
+	return newConfig, err
```

Comment:
You should be setting 
config=newConfig

Then a call to config.Default() will get the latest config.

Reply:
config is a global that eveyone gets after the first call to Default()

Reply:
fixed.